examples/nodejs/src/escrow-flow.ts

```diff
@@ -0,0 +1,124 @@
+import arg from "arg";
```

Comment:
I like this example... It should be well visible, maybe a FAQ ?  Like `I want to use Marlowe with NodeJS or/and Deno ? How can I do it ?`

Reply:
eventually yeah! that is why I added @nstanford5 to review this as well